config/webpack.config.babel.js

```diff
@@ -47,6 +48,7 @@ const WEBPACK_DEV_SERVER_USE_TLS = process.env.WEBPACK_DEV_SERVER_USE_TLS === 't
 const TTN_LW_TLS_CERTIFICATE = process.env.TTN_LW_TLS_CERTIFICATE || './cert.pem'
 const TTN_LW_TLS_KEY = process.env.TTN_LW_TLS_KEY || './key.pem'
 const TTN_LW_TLS_ROOT_CA = process.env.TTN_LW_TLS_ROOT_CA || './cert.pem'
+const USE_SENTRY_SOURCEMAPS = Boolean(process.env.SENTRY_DSN)
```

Comment:
```suggestion
const WEBPACK_UPLOAD_SENTRY_SOURCEMAPS = Boolean(process.env.SENTRY_DSN) && process.env.WEBPACK_UPLOAD_SENTRY_SOURCEMAPS === 'true'
const TTN_LW_TLS_CERTIFICATE = process.env.TTN_LW_TLS_CERTIFICATE || './cert.pem'
const TTN_LW_TLS_KEY = process.env.TTN_LW_TLS_KEY || './key.pem'
const TTN_LW_TLS_ROOT_CA = process.env.TTN_LW_TLS_ROOT_CA || './cert.pem'
```

I think it's better to set this explicitly, otherwise we might end up accidentally uploading faulty source maps from local machines.

Reply:
Just noticed this stuff for the first time. I don't remember any documentation saying that webpack config only supports environment variables (and not the config file)

Reply:
Hmm. That's true. These are all development related though (changing behavior of the dev server or CI) and none of those settings are needed for production. 

Would it be acceptable for you if we document this constraint, or should we take time to source the config file as well?

Reply:
I'm fine with documenting it

Reply:
@pgalic96 can you take care as part of updating the development documentation for the new environment variables? A simple sentence stating that these configs cannot be source from the config file should suffice.